Library/Homebrew/cmd/gist-logs.rb

```diff
@@ -40,6 +47,19 @@ def gistify_logs(f)
     puts url if url
   end
 
+  def brief_build_info(f)
+    build_time_str = f.logs.ctime.strftime("%Y-%m-%d %H:%M:%S")
+    hostname = Socket.gethostname
+    s = <<-EOS.undent
+      Homebrew build logs for #{f.name} on #{MACOS_FULL_VERSION}
```

Comment:
Similar as above:
* `f.full_name` instead of `f.name`.
* `OS_VERSION` instead of `MACOS_FULL_VERSION`.